Build error during openapi code generation

```
$make generate-openapi
Generating OpenAPI structs for Kubernetes Clients
./build/codegen --generator-version 36ebc4887cdc openapi --output-package=pkg/client --input-package=github.com/jenkins-x/jx/v2/pkg/apis --group-with-version=jenkins.io:v1 --version=2.0 --module-name=github.com/jenkins-x/jx/v2 --semver=v2
INFO[0000] installing k8s.io/kube-openapi/cmd/openapi-gen with version 36ebc4887cdc via 'go get' to /Users/hardy/.jx/codegen/go  app=codegen
error installing kubernetes openapi tools: error running go get -u k8s.io/kube-openapi/cmd/openapi-gen@36ebc4887cdc, output go: finding k8s.io 36ebc4887cdc
go: finding k8s.io/kube-openapi 36ebc4887cdc
go: finding k8s.io/kube-openapi/cmd/openapi-gen 36ebc4887cdc
go: finding k8s.io/kube-openapi/cmd 36ebc4887cdc
go: finding github.com/modern-go/concurrent latest
go: finding k8s.io/gengo latest
go: finding github.com/PuerkitoBio/urlesc latest
go: finding golang.org/x/net latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/xerrors latest
# k8s.io/klog/v2
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:690:45: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:702:43: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:706:48: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:721:44: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:739:55: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:764:32: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:881:43: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:1234:10: undefined: logr.InfoLogger: failed to run 'go get -u k8s.io/kube-openapi/cmd/openapi-gen@36ebc4887cdc' command in directory '', output: 'go: finding k8s.io 36ebc4887cdc
go: finding k8s.io/kube-openapi 36ebc4887cdc
go: finding k8s.io/kube-openapi/cmd/openapi-gen 36ebc4887cdc
go: finding k8s.io/kube-openapi/cmd 36ebc4887cdc
go: finding github.com/modern-go/concurrent latest
go: finding k8s.io/gengo latest
go: finding github.com/PuerkitoBio/urlesc latest
go: finding golang.org/x/net latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/xerrors latest
# k8s.io/klog/v2
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:690:45: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:702:43: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:706:48: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:721:44: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:739:55: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:764:32: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:881:43: undefined: logr.InfoLogger
/Users/hardy/.jx/codegen/go/pkg/mod/k8s.io/klog/v2@v2.1.0/klog.go:1234:10: undefined: logr.InfoLogger'
make: *** [codegen-openapi] Error 1
```

See also pull request #7334 and #7328 where the error appeared for the first time.

--- a/cmd/codegen/generator/client_set.go
+++ b/cmd/codegen/generator/client_set.go
@@ -40,7 +40,7 @@ func InstallCodeGenerators(version string, gopath string) error {
 	}
 	path := fmt.Sprintf("%s/...", basePath)
 	util.AppLogger().Infof("installing %s version %s into %s", path, version, gopath)
-	err := util.GoGet(path, version, gopath, true, false, true)
+	err := util.GoGet(path, version, gopath, true, false, false)
 	if err != nil {
 		return err
 	}

--- a/cmd/codegen/generator/open_api.go
+++ b/cmd/codegen/generator/open_api.go
@@ -166,7 +166,7 @@ type schemaWriterTemplateData struct {
 // InstallOpenApiGen installs the openapi-gen tool from the github.com/kubernetes/kube-openapi repository.
 func InstallOpenApiGen(version string, gopath string) error {
 	util.AppLogger().Infof("installing %s with version %s via 'go get' to %s", openApiGen, version, gopath)
-	err := util.GoGet(openApiGen, version, gopath, true, false, true)
+	err := util.GoGet(openApiGen, version, gopath, true, false, false)
 	if err != nil {
 		return err
 	}

